talisker/requests.py

```diff
@@ -85,9 +85,19 @@ class RequestsMetric:
     )
 
 
-def register_ip(ip, name):
-    """Register a human friendly name for an IP address for metrics."""
-    HOSTS[ip] = name
+def register_endpoint_name(endpoint, name):
+    """Register a human friendly name for an IP:PORT address for metrics."""
+    parsed = parse_url(endpoint)
+    HOSTS[parsed.netloc] = name
+
+
+# backwards compat alias
+register_ip = register_endpoint_name
```

Comment:
can we wrap this in a decorator like https://python-deprecated.readthedocs.io/en/latest/tutorial.html ?